### ISSUE TITLE
[Bug][Windows x86 Installer] Plugin manager is not present/installed.

### ISSUE THREAD

**Issue by Bigfootmech** [notepad-plus-plus/notepad-plus-plus#3841]
### Description of the Issue
In versions 7.3 and below, plugin manager was distributed with the main release, and it still says it is on the website.

https://notepad-plus-plus.org/resources.html

However, in the newest few releases, it's not actually installed, or in the install menu as an option.

### Steps to Reproduce the Issue
1. Download 7.5.1 Notepad++ Installer 32-bit x86
2. Run executable
3. Click Ok (for english)
4. Next -> I agree -> Next
5. Press [+] to expand plugins submenu for installation
6. Plugin manager is not there.

7. Additionally, if all traces of N++ are removed from a machine, and installer completes, the final version will not have plugin manager as a plugin.

However, if you install an old version, then install a new one over it, the plugin manager from the old distribution stays as a plugin.

### Expected Behavior
Plugin Manager plugin is included in Notepad++ release.

### Actual Behavior
Plugin manager does not appear in the install menu, and is not installed.

**Comment by sasumner** [comment]
It was a design decision to remove Plugin Manager from v7.5 onwards.

**Comment by Bigfootmech** [comment]
Then the bug is in the https://notepad-plus-plus.org/resources.html page Plugins section
As well as possibly others? (including downloads page, referring to only x64 not coming with plugin manager)

**Comment by sasumner** [comment]
If we are going to make Issues out of website errors then we are really in trouble as the website is out-of-date in many, MANY places.  To think about it, with 1800+ issues we are already in serious trouble.

**Comment by sasumner** [comment]
Duplicate of #3848

**Comment by rcdailey** [comment]
I have to be blunt here. I think it was a poor decision to remove the Plugin Manager without a suitable replacement. AFAIK, there is no longer a built-in mechanism to install, remove, and update plugins. This creates a serious usability issue with Notepad++.

For months now, I have had to avoid upgrading until a replacement is implemented. I haven't kept up with release notes, so I don't know if a replacement has been provided yet. But that's part of the reason I stumbled upon this issue is that I'm trying to figure that out.

**Comment by movedoa** [comment]
What is the status on the new plugin manager? Is there a ETA?

**Comment by SinghRajenM** [comment]
New plugin manager is about to release may be in 7.6.

**Comment by MetaChuh** [comment]
the new built in `plugins admin` is available since notepad++ 7.6.

please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.

thank you for your help and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)
